test/helpers/patient_query_helper_test.rb

```diff
@@ -258,6 +258,75 @@ class PatientQueryHelperTest < ActionView::TestCase
 
```

Comment:
can you also add to the frontend test that changes number inputs an section changing to blank? test is on line 600

Reply:
also correctly rendering the blank date option

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1252/commits/139bc7d564fd6b8256a9c0f121518989e083bac6